doc/chapters/getting-started/examples/09_setcapabilities.sh

```diff
@@ -3,6 +3,5 @@ gdbus call --system \
 --object-path /com/pelagicore/SoftwareContainerAgent \
 --method com.pelagicore.SoftwareContainerAgent.SetCapabilities \
 0 \
-"['com.pelagicore.temperatureservice.gettemperature',
-'com.pelagicore.temperatureservice.settemperature']"
+"['com.pelagicore.temperatureservice.settemperature']"
```

Comment:
Nein!

Seriously though... not sure if it serves any purpose for the example. Do you still want to have it?